docs/explanation.rst

```diff
@@ -0,0 +1,121 @@
+Explanantion
+=============
+
+..
+  This section requires more improvements.
```

Comment:
This comment does not add any value. With this pull request, you improve the quality of the documentation already so much. If you think you want to improve it even further, you could create an issue at the tracker, or just do it whenever you feel like.

Reply:
I'll remove the comment. 

But there is many causes make me unsatisfied with this section even after my edit:

- Titles is not exacltly the subject below them.
- Th whole section isn't logically built ( its titles not arranges neither chronologically nor  from easy to hard )
- The `default_dest` parameter, I can't really understand why it exist here?. In this section I kept the old subject with minimal arrangement of the sentenses.

All of this causes I will not try to fix them in this pull request. As I need to open `issues` before trying to touch them.

Reply:
There is many causes make me unsatisfied with this section even after my edit:

- Titles is not exacltly the subject below them.
- Th whole section isn't logically built ( its titles not arranges neither chronologically nor  from easy to hard )
- The `default_dest` parameter, I can't really understand why it exist here?. In this section I kept the old subject with minimal arrangement of the sentenses.

All of this causes I will not try to fix them in this pull request. As I need to open `issues` before trying to touch them.